recipes/wix/meta.yaml

```diff
@@ -0,0 +1,26 @@
+{% set version = "3.10.3" %}
+
+package:
+  name: wix
+  version: {{ version }}
+
+source:
+  path: ./src
+
+build:
+  number: 0
+  skip: true  # [not win]
+
+requirements:
+  run:
+   - vc  14
+
+about:
+  home: http://wixtoolset.org/
+  summary: WiX Toolset
+  license: MS-RL
+  license_file: COPYING
```

Comment:
This will pull it from the source directory. So it is already working without having the license file in the recipe.